go-apps/meep-mg-manager/server/mg-manager.go

```diff
@@ -319,12 +367,21 @@ func clearScenario() {
 func processScenario(model *mod.Model) error {
 	log.Debug("processScenario")
 
+	// Reset service info maps
+	mgm.svcInfoMap = make(map[string]*serviceInfo)
+	mgm.mgSvcInfoMap = make(map[string]*mgServiceInfo)
+	mgm.mgList = make(map[string]bool)
```

Comment:
- If the above maps share the same keys - it may be beneficial to maintain a single map.
- `mgList` is a map

Reply:
You are correct. I removed mgList and now use mgSvcInfoMap instead.